### DISCUSSION TITLE
Splitanzeige auch ohne eAuto / Wärmepumpe

### DISCUSSION THREAD

**Discussion by Zeppelin-PV** [solectrus/solectrus#3597] [Ideas]
hallo Georg
seit der Teilnahme an der Beta summarizer habe ich in den Langzeitgraphen (Woche-->Gesamt) ebenfalls eine gesplittete Balkenanzeige
Bei mir wird aber immer zu 100% PV-Anteil angezeigt. Nun (nochmals) die Frage -
- wäre es für einen "Otto-Normal-Nutzer" von Solectrus, ohne eAuto und/oder Wärmepumpe nicht auch sinnvoll zu sehen wie hoch der Anteil von Netzbezug im Hausverbrauch ausmacht ? Vor allem in der dunklen Jahreszeit ?
- habe mit Verlaub ein Screenshot von Deiner Liveseite gemacht
![grafik](https://github.com/user-attachments/assets/a8203610-1efe-4a2a-b943-a2c97139fd67)

dasselbe von meiner PV
![grafik](https://github.com/user-attachments/assets/8eaf2fd4-5cd1-41a8-81df-0dc1a4bdaadd)

**Comment by ledermann** [comment]
Läuft bei dir der Power-Splitter? Das wäre eine Erklärung für den fehlenden Netzbezug im Diagramm.

**Reply by Zeppelin-PV** [reply]
ich denke nicht, 
müsste doch zu sehen sein bei docker compose pull - oder ???
habe noch die env und compose angefügt

shelly-collector-therme Skipped - Image is already being pulled by shelly-collector-aufzug                                                  0.0s 
shelly-collector-terrasse Skipped - Image is already being pulled by shelly-collector-aufzug                                                0.0s 
postgresql Pulled                                                                                                                           1.2s 
forecast-collector Pulled                                                                                                                   0.7s 
dashboard Pulled                                                                                                                            4.3s 
cf04c63912e1 Already exists                                                                                                               0.0s 
79810d6b1dd4 Already exists                                                                                                               0.0s 
57a01e81a397 Already exists                                                                                                               0.0s 
3f2e659c57c3 Already exists                                                                                                               0.0s 
c74ce8633db9 Already exists                                                                                                               0.0s 
f7a635f76e3d Pull complete                                                                                                                0.8s 
c5ed41442b0c Pull complete                                                                                                                0.9s 
ac2843d13509 Pull complete                                                                                                                0.9s 
8bee1cba96fe Pull complete                                                                                                                2.3s 
d40a493b0a28 Pull complete                                                                                                                3.2s 
5056f46bfeee Pull complete                                                                                                                3.2s 
watchtower Pulled                                                                                                                           1.2s 
senec-collector Pulled                                                                                                                      0.7s 
influxdb Pulled                                                                                                                             1.2s 
shelly-collector-aufzug Pulled                                                                                                              0.7s 
redis Pulled                                                                                                                                1.2s 

Container solectrus-watchtower-1                 Running                                                                                    0.0s 
Container solectrus-redis-1                      Healthy                                                                                    0.8s 
Container solectrus-influxdb-1                   Healthy                                                                                    0.8s 
Container solectrus-senec-collector-1            Running                                                                                    0.0s 
Container solectrus-postgresql-1                 Healthy                                                                                    0.8s 
Container solectrus-dashboard-1                  Started                                                                                    1.1s 
Container solectrus-shelly-collector-aufzug-1    Running                                                                                    0.0s 
Container solectrus-forecast-collector-1         Running                                                                                    0.0s 
Container solectrus-shelly-collector-terrasse-1  Running                                                                                    0.0s 
Container solectrus-shelly-collector-therme-1    Running                                                                                    0.0s 

[.env.txt](https://github.com/user-attachments/files/17548605/default.env.txt)
[compose.yaml.txt](https://github.com/user-attachments/files/17548606/compose.yaml.txt)

**Reply by Zeppelin-PV** [reply]
auch unter docker compose ps ist nichts von power-splitter zu sehen

admin@raspberrypi:~/solectrus $ docker compose ps
NAME                                    IMAGE                                         COMMAND                  SERVICE                     CREATED        STATUS                  PORTS
solectrus-dashboard-1                   ghcr.io/solectrus/solectrus:pr-3533           "docker/entrypoint.sâ€¦"   dashboard                   11 hours ago   Up 11 hours (healthy)   0.0.0.0:3000->3000/tcp, :::3000->3000/tcp
solectrus-forecast-collector-1          ghcr.io/solectrus/forecast-collector:latest   "bundle exec app/maiâ€¦"   forecast-collector          4 days ago     Up 4 days               
solectrus-influxdb-1                    influxdb:2.7-alpine                           "/entrypoint.sh inflâ€¦"   influxdb                    4 days ago     Up 4 days (healthy)     0.0.0.0:8086->8086/tcp, :::8086->8086/tcp
solectrus-postgresql-1                  postgres:16-alpine                            "docker-entrypoint.sâ€¦"   postgresql                  4 days ago     Up 4 days (healthy)     5432/tcp
solectrus-redis-1                       redis:7-alpine                                "docker-entrypoint.sâ€¦"   redis                       4 days ago     Up 4 days (healthy)     6379/tcp
solectrus-senec-collector-1             ghcr.io/solectrus/senec-collector:latest      "bundle exec app.rb"     senec-collector             4 days ago     Up 4 days               
solectrus-shelly-collector-aufzug-1     ghcr.io/solectrus/shelly-collector:latest     "bundle exec app.rb"     shelly-collector-aufzug     4 days ago     Up 4 days               
solectrus-shelly-collector-terrasse-1   ghcr.io/solectrus/shelly-collector:latest     "bundle exec app.rb"     shelly-collector-terrasse   4 days ago     Up 4 days               
solectrus-shelly-collector-therme-1     ghcr.io/solectrus/shelly-collector:latest     "bundle exec app.rb"     shelly-collector-therme     4 days ago     Up 4 days               
solectrus-watchtower-1                  containrrr/watchtower                         "/watchtower --scopeâ€¦"   watchtower                  4 days ago     Up 4 days (healthy)     8080/tcp

**Reply by ledermann** [reply]
Sorry, da habe ich dich versehentlich auf die falsche Fährte gebracht. Natürlich ist kein Power-Splitter zu finden, der benötigt ja E-Auto und/oder Wärmepumpe.

Die fehlerhafte Darstellung habe ich soeben korrigiert. Wenn keine Power-Splitter-Werte vorhanden sind, erfolgt jetzt auch keine Splitdarstellung im Diagramm.

Deinen Vorschlag, bei Nur-Hausverbrauch dennoch eine Splitdarstellung zu erreichen (durch Heranziehen der Autarkie), werde ich mal prüfen.

**Reply by Zeppelin-PV** [reply]
Korrektur, fehlerhafte Darstellung wurde gerade gezogen,
... _werde ich mal prüfen_.
bitte dranbleiben

**Reply by ledermann** [reply]
Fortführung in #3608
